integration-tests/infinispan-embedded/pom.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-integration-tests-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-integration-test-infinispan-embedded</artifactId>
+    <name>Quarkus - Integration Tests - Infinispan Embedded</name>
+    <description>Module that contains Infinispan Embedded related tests</description>
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-infinispan-embedded</artifactId>
+        </dependency>
+
+        <!-- Allow some basic support for @Inject (required for JAXRS) -->
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-arc</artifactId>
+        </dependency>
+        <!-- .. and some REST endpoints -->
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy</artifactId>
+        </dependency>
+        <!-- Test Dependencies - Note this is not test scope as native compilation doesn't use test classes/dependencies -->
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-narayana-jta</artifactId>
```

Comment:
should the `quarkus-narayana-jta` not be a mandated dependency? Can it run without it?

Reply:
Sure, if you don't have any transactional caches it works just fine. But now that I know I should force users to only use Quarkus transaction manager I can add some more logic around this and some checks.

Reply:
Please verify: does native compilation still work fine whtn the TXM is missing? I bet it won't compile ;)

Reply:
It passes fine when I comment out this line and the `testTransactionRolledBack` test that failed without it.

```
INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.426 s - in io.quarkus.it.infinispan.embedded.InfinispanClientFunctionalityInGraalITCase
```